storage/local/chunk/chunk.go

```diff
@@ -27,21 +27,30 @@ import (
 	"github.com/prometheus/prometheus/storage/metric"
 )
 
-// DefaultChunkEncoding can be changed via a flag.
-var DefaultChunkEncoding = DoubleDelta
+// ChunkLen is the length of a chunk in bytes.
+const ChunkLen = 1024
```

Comment:
Yeah, I found that a bit too radical TBH, but will change it in case you insist :)